scripts/Muon/GUI/Common/contexts/muon_context.py

```diff
@@ -214,3 +214,44 @@ def dead_time_table(self, run):
             return self.data_context.get_loaded_data_for_run(run)["DataDeadTimeTable"]
         elif self.gui_context['DeadTimeSource'] == 'None':
             return None
+
+    def get_names_of_workspaces_to_fit(self, runs='', group_and_pair='', phasequad=False, rebin=False):
+        if group_and_pair == 'All':
+            group = self.group_pair_context.group_names
+            pair = self.group_pair_context.pair_names
+        else:
+            group_pair_list = group_and_pair.replace(' ', '').split(',')
+            group = [item for item in group_pair_list if item in self.group_pair_context.group_names]
```

Comment:
would this be clearer as `groups =[group for group in group_pair_list if group in self.group_pair_context.group_names]`

I know it could be a pair but that would fail the if.